back/src/test/java/com/cocktailpick/back/user/docs/UserDocumentation.java

```diff
@@ -25,4 +25,14 @@ public static RestDocumentationResultHandler findMe() {
 			)
 		);
 	}
+
+	public static RestDocumentationResultHandler updateUser() {
+		return document("user/me",
+			requestHeaders(
+				headerWithName("authorization").description("Bearer 토큰")),
+			responseFields(
```

Comment:
```suggestion
			requestFields(
```
UserController에서 updateCurrentUser 메서드는 `@RequestBody`만 있고, 반환하는 `@ResponseBody`는 없어서 `responseFields`를 `requestFields`로 수정해주셔야 할 것 같아요! :)

Reply:
역시 API 문서의 대가 두강

Reply:
네 제거하였습니다 :)

Reply:
ㄷㄷ 저걸 어떻게 발견했지